es6-destructuring/main.js

```diff
@@ -0,0 +1,60 @@
+const book1 = {
+  title: 'Goodnight Punpun',
+  author: 'Inio Asano',
+  libraryID: 3353,
+};
+
+const { title: title1 } = book1;
```

Comment:
The instructions say to do this first one "with the same names as book1's property names".